.env

```diff
@@ -1,3 +1,3 @@
-REACT_APP_DISCOVER=chance-source-liquid-nelson
-REACT_APP_VERSION=$npm_package_version
-REACT_APP_ENVIRONMENT=prod
+VITE_DISCOVER=chance-source-liquid-nelson
+VITE_VERSION=$npm_package_version
```

Comment:
I don't think this works? Does it?

Reply:
You're correct. It doesn't work. I implemented a different solution.